gomatic/go_cd_configurator.py

```diff
@@ -258,7 +263,13 @@ def ensure_replacement_of_elastic(self):
 
     @property
     def elastic(self):
-        return Elastic(self.__server_element_ensurance().element.find('elastic'))
+        gocd_version_with_elastic_outside_server_tag = 18
+        gocd_major_version = int(self.server_version.split('.')[0])
+        if gocd_major_version < gocd_version_with_elastic_outside_server_tag:
+            elastic_element = Elastic(Ensurance(self.__xml_root).element.find('elastic'))
+        else:
+            elastic_element = Elastic(self.__server_element_ensurance().element.find('elastic'))
```

Comment:
It seems line 269 and 271 are swapped here? If I swap them, the tests pass.

Reply:
@dudadornelles You're right. I will amend this commit with the lines swapped. :)